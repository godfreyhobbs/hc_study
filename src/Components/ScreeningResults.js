import React, { Component } from 'react';
import logo from '../logo.svg';

class ScreeningResults extends Component {
  render() {
    return (
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <h1 className="App-title">Screening Results</h1>
      </header>
    );
  }
}
export default ScreeningResults;
