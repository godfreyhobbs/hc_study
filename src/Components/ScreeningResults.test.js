import React from 'react';
import ReactDOM from 'react-dom';
import ScreeningResults from './ScreeningResults';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ScreeningResults />, div);
  ReactDOM.unmountComponentAtNode(div);
});



it('renders correctly', () => {
  const tree = renderer
.create(<ScreeningResults />)
   .toJSON();
expect(tree).toMatchSnapshot();
});
