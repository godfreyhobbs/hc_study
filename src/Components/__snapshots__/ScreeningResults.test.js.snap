// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    className="App-title"
  >
    Screening Results
  </h1>
  <p>
    Congratulations! You are eligible for this study.
  </p>
  <button
    onClick={[Function]}
  >
    Start Questionnaire
  </button>
</div>
`;
